build.sbt

```diff
@@ -117,8 +109,8 @@ def playJsonOps(scalacVersion: String, includePlayVersion: String): Project = {
           scalaCheckOps(scalaCheckVersion),
           scalaTest(scalaCheckVersion)
         ) ++ {
-          includePlayVersion match {
-            case Play_2_7 => Seq(
+          scalaCheckVersion match {
+            case ScalaCheck_1_14 => Seq(
               scalaTestPlusScalaCheck(scalaCheckVersion)
```

Comment:
The addition of this library is required when using the latest ScalaTest, and the choice of ScalaTest is based on the version of ScalaCheck, not Play.